moped-database/views/component_arcgis_online_view.sql

```diff
@@ -1,16 +1,6 @@
--- Most recent migration: moped-database/migrations/1733184698645_add_comp_work_types/up.sql
+-- Most recent migration: moped-database/migrations/1737754622032_deprecate_type_tables/up.sql
 
-CREATE OR REPLACE VIEW component_arcgis_online_view AS WITH work_types AS (
```

Comment:
Since we now have this calculated upstream in the `project_list_view` that is joined into this view, I updated to select the component work types from there. This CTE is no longer needed.